src/GrpcProtos/capture.proto

```diff
@@ -382,6 +382,13 @@ message CGroupMemoryUsage {
   int64 active_file_bytes = 12;
 }
 
+message MemoryEventWrapper {
+  uint64 timestamp_ns = 1;
```

Comment:
This is intended. The timestamp in a specific memory event (e.g., `SystemMemoryUsage`, `CGroupMemoryUsage`, or `ProcessMemoryUsage`) records the time when this memory event is actually collected. 

On one hand, for events collected in the same sampling window, we can check whether the difference between those timestamps is small enough in MemoryTracingIntegrationTest; on the other hand, they are recorded in case we need the more precise timestamp information.